pkg/cmd/generate.go

```diff
@@ -64,15 +69,18 @@ func RunGenerate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	// print w/ escaped newlines
+	// Add separator to stringify output
+	stringOutput := "'" + strings.Join(output, "\n\n +----------------------------------------------------------+ \n\n")
```

Comment:
move this functionality to the `PrintOrWriteOut` function

Reply:
All the stuff between here and the actual call to `PrintOrWriteOut` should be placed in that function

Reply:
Updated